### DISCUSSION TITLE
大干30天！郑州宣战“烂尾楼”：确保全面复工 资金来源也确定！你们相信吗？

### DISCUSSION THREAD

**Discussion by caotianchao** [WeNeedHome/SummaryOfLoanSuspension#1174] [General]
【大干30天！郑州宣战“烂尾楼”：确保全面复工 资金来源也确定！】9月7日，据大河报报道，郑州市于近日印发了《郑州市“大干30天，确保全市停工楼盘全面复工”保交楼专项行动实施方案》的通知。通知要求，10月6日前，实现郑州全市所有停工问题楼盘项目全面持续实质性复工，并提出五方面具体实施方案。

**Comment by oceanzhang860411** [comment]
不給錢，誰幫你開工？等著看戲吧

**Reply by caotianchao** [reply]
没错

**Comment by xijinping0** [comment]
“不管你信不信，反正我是信了。”

**Comment by Ventery** [comment]
然后烂尾楼问题解决了，🐸🐸们就失忆了一般，开始寻找新黑点。全然忘记自己同胞被卖到柬埔寨的悲惨结局😄😄😄😄😄

**Reply by oceanzhang860411** [reply]
什麼事不能解決？ 100年後，這代人都死了也就解決了。

**Reply by itsoursduty** [reply]
不会真有人相信能解决吧

**Comment by Cegazhos** [comment]
荷兰没有部门出来认这个文件，中修也没出来认。
你应该写上这一点

**Comment by cccp057** [comment]
造楼可以免费造吗？计划经济或许可以，市场经济是要出钱的，钱从哪来？就看郑州3月卖房那猴急的样子就知道不大可能了

**Reply by Cegazhos** [reply]
只能财政兜底先出了复工的钱。交房后收了尾数 或许还不亏。

**Comment by Dreaming-Pepper** [comment]
有文件原件图片或电子版么，贴来康康嗄，我浅搜了下没有唉。
30天持续复工，钱都没有，做啥梦这帮人。
（嗑瓜子）

**Comment by diaohancai** [comment]
二十大之前的作秀吧，二十大之前估计是复工状态的，过了之后就...

**Comment by 505Games-oss** [comment]
我不信  除非它用开水烫自己小鸡鸡

**Comment by NecZhang** [comment]
大干三十天验收倒计时

**Reply by Cegazhos** [reply]
> 大干三十天验收倒计时

已经结束辣
